apps/meteor/app/cloud/server/functions/getCheckoutUrl.ts

```diff
@@ -21,9 +28,8 @@ export const getCheckoutUrl = async (token: string, body: CheckoutBodyParams) =>
 			body,
 		});
 
-		if (!response.url) {
-			const { i18n } = await response.json();
-			throw new Error(i18n);
+		if (!response.ok) {
+			throw new Error(await response.json());
```

Comment:
@ggazzo I could not reproduce a scenario that got this error, because I didn't find a `ok` attribute in the payload ... Is this going to work? Maybe I've missed something